src/core/server/graph/schema/schema.graphql

```diff
@@ -564,6 +564,11 @@ enum FEATURE_FLAG {
   """
   COMMENT_SEEN
 
+  """
+  Z_KEY will allow Z key traversal and the mobile toolbar.
+  """
+  Z_KEY
```

Comment:
Since this key may change in the future, it's a bit funny we're bringing SBN-isms into Coral. Would have maybe called this `ENABLE_KEYBOARD_COMMENT_TRAVERSAL` or something that describes what `Z_KEY` is doing. But the comment will suffice for now. We can fix this later.